R/utils.R

```diff
@@ -83,6 +83,22 @@ git_init <- function(path = ".") {
   )
 }
 
+#' Provide a singular interface for hook calls to docopt
+#'
+#' docopt provides different processing for a single string
+#' than an array/vector, and so this function wraps docopt
+#' to
```

Comment:
Is this sentence finished?

Reply:
Huh.  Not sure where the rest of that went.